### PR TITLE
wording

### PR DESCRIPTION
**Pull request by remicollet**
Just checking diff between PHP 3.0 and 3.01 License and on which one is Xdebug license based
found minor wording improvement

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -------------------------------------------------------------------- 
-                 The Xdebug License, version 1.02
-             (Based on "The PHP License", version 3.0)
-   Copyright (c) 2003-2018 Derick Rethans. All rights reserved.
+                 The Xdebug License, version 1.03
+             (Based on "The PHP License", version 3.01)
```
Comment (remicollet) [inline LICENSE:3]: IIRC, some distro (deb) have issue with 3.0

File: LICENSE [hunk 2]
```diff
@@ -35,8 +35,8 @@ are met:
      applicable to covered code created under this License.
 
   6. Redistributions of any form whatsoever must retain the following
-     acknowledgment: "This product includes Xdebug, freely available
-     from <https://xdebug.org/>".
+     acknowledgment: "This product includes Xdebug software, freely
```
Comment (remicollet) [inline LICENSE:38]: "software" word add in PHP 3.0.1

File: LICENSE [hunk 3]
```diff
@@ -1,7 +1,7 @@
 -------------------------------------------------------------------- 
-                 The Xdebug License, version 1.02
-             (Based on "The PHP License", version 3.0)
-   Copyright (c) 2003-2018 Derick Rethans. All rights reserved.
+                 The Xdebug License, version 1.03
+             (Based on "The PHP License", version 3.01)
+   Copyright (c) 2003-2022 Derick Rethans. All rights reserved.
```
Comment (remicollet) [inline LICENSE:4]: Perhaps simpler to drop (as in PHP) the date range

Comment (derickr) [reply LICENSE:4]: The date range here is for the copyright of the *license*, and not the code. I don't care either way though. I do sort of like having the earliest year in it :-).

File: LICENSE (hunk 1 above)
Comment (derickr) [reply LICENSE:3]: Yeah, that's fine... pointless IMO, but fine :-)

File: LICENSE (hunk 2 above)
Comment (derickr) [reply LICENSE:38]: OK. Was that really why 3.0 was not OK? Or was it the line for Zend Engine (that I of course don't have).

**Comment by derickr** [comment]
I've merged this, but your PR has a spurious commit in it that GIT locally removed (correctly). So it seems I have to close this manually. Thanks!

**Pull request closed without merging**